### PR TITLE
#1094 Fix initialization of Modelica combiTable when an initial state…

### PR DESCRIPTION
**Pull request by rosiereflo**
… is loaded

### CODE REVIEW

**Comment by rosiereflo** [comment]
Sonar failure is caused by code duplication so it can be ignored.

File: util/envDynawo.sh [hunk 1]
```diff
@@ -1304,8 +1304,10 @@ nrt() {
   if ! is_launcher_installed; then
     install_launcher || error_exit "Error during launcher installation."
   fi
+  pushd "$DYNAWO_HOME" > /dev/null
```
Comment (MarianneSaugierRTE) [inline util/envDynawo.sh:1307]: Why did you need to update this file with this development?

**Review by MarianneSaugierRTE — APPROVED** [review]

File: util/envDynawo.sh (hunk 1 above)
Comment (rosiereflo) [reply util/envDynawo.sh:1307]: This is not related to the development but was useful for my validation: CombiTable using files as inputs use paths relative to DYNAWO_HOME, so from version-validation or if you launch the nrt from another path than DYNAWO_HOME some tests were failing. Those lines make sure that the current path is dynawo home when nrt are launched to avoid those false positives

Comment (MarianneSaugierRTE) [reply util/envDynawo.sh:1307]: ok thanks for the explanation :)
